pymllm/orchestrator/scheduler_process.py

```
@@ -55,6 +56,27 @@
+# Brief poll timeout (ms) used between decode batches to avoid 100% CPU spin.
+# 1 ms is enough to yield the CPU core to the OS scheduler while adding
+# negligible latency (decode steps typically take >1 ms on the GPU anyway).
+# Override via MLLM_DECODE_POLL_TIMEOUT_MS env var for testing.
+def _read_decode_poll_timeout_ms() -> int:
+    raw = os.environ.get("MLLM_DECODE_POLL_TIMEOUT_MS", "1")
+    try:
+        val = int(raw)
+    except ValueError:
+        raise ValueError(
+            f"MLLM_DECODE_POLL_TIMEOUT_MS must be a non-negative integer, got {raw!r}"
+        )
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)